user_manual.adoc

```diff
@@ -342,6 +342,26 @@ If you look at the running config, the first time the client successfully
 connects to the server a variable `Client.server_serial_number` will be written
 to the config. If that exists, the client successfully made a connection.
 
+Importing the NSRL
+~~~~~~~~~~~~~~~~~~
+The National Software Registry List (NSRL) is a collection of known software managed by NIST. It is commonly used in forensics to reduce the scope of analysis of already known software. This is typically done by whitelisting anything on the NSRL by hash. 
+
+GRR has the ability to import the NSRL. This function prepopulates the GRR datastore with all known hashes and reduces the need for GRR to collect these from the client systems. This can be done by downloading the latest quarterly release of the NSRL from Link:http://www.nsrl.nist.gov/Downloads.htm#isos[NIST].
+
+1. Download NSRL from NIST
+2. Expand the zipped file containing hashes
+3. Run "import_nsrl_hashes.py" with the appropriate configuration options
+
+----
+~/grr/tools# python import_nsrl_hashes.py --config /etc/grr/grr-server.yaml --secondary_configs /etc/grr/server.local.yaml --filename /media/<path to expanded NSRL>/NSRLFile.txt
```

Comment:
Thanks for this PR! One quick question, the secondary config you give here as a parameter should be automatically loaded since it's the Config.writeback. Do you actually need to specify it? If so, that would be a bug. Please remove the --secondary_configs flag for the example.